rebasehelper/checkers/rpmdiff_tool.py

```diff
@@ -151,11 +152,34 @@ def run_check(cls, results_dir, **kwargs):
                 lines.append('Following files were %s:' % key)
                 lines.extend(val)
 
-        rpmdiff_report = os.path.join(cls.results_dir, 'report-' + cls.CMD + '.log')
+        rpmdiff_report = os.path.join(cls.results_dir, cls.CMD + '.txt')
+
+        counts = {'added': len(results_dict['added']),
+                  'removed': len(results_dict['removed']),
+                  'changed': len(results_dict['changed'])}
+
         try:
             with open(rpmdiff_report, "w") as f:
                 f.write('\n'.join(lines))
         except IOError:
             raise RebaseHelperError("Unable to write result from %s to '%s'" % (cls.CMD, rpmdiff_report))
 
-        return {rpmdiff_report: None}
+        return {'path': cls.get_checker_output_dir_short(), 'files_changes': counts}
+
+    @classmethod
+    def format(cls, data):
+        """
+        Format rpmdiff output for outputtool
+        :param data: rpmdiff output data dictionary
+        :return: string formated output
+        """
+        output_string = cls.get_underlined_title("rpmdiff")
+
+        output_string += " - %s added files\n" % data['files_changes']['added']
```

Comment:
Newline character moved to the end of string in each of checkers `format` method, as that is more human readable I suggest.